wire-ios/Wire-iOS/Sources/UserInterface/Calling/CallViewController/CallViewController.swift

```diff
@@ -534,12 +538,6 @@ extension CallViewController: ConstantBitRateAudioObserver {
 
 }
 
-extension CallViewController: NetworkQualityObserver {
```

Comment:
Just to check my understanding, we are removing the observer here because the VC if for 1:1 class right?

Reply:
`CallViewController` is the root VC of the calling UI. It has two child VCs: `CallGridViewController` & `CallInfoRootViewController`. Both child VCs have a `configuration` property which, when set, triggers an update of their views. This is what `updateConfiguration()` does: it recreates and sets the configuration objects for each child VC.

By removing the observer from `CallViewController` and moving it to `CallGridViewController` we avoid going through this main pipeline of UI updates through configuration objects. Instead, we're directly targeting the UI components that need to be updated

Reply:
That makes sense. Thanks for the explanation!